### PR TITLE
chore(DB): Fix SQL imports/database revision

### PR DESCRIPTION
**Pull request by Nyeriah**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Kitzunu — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1636206943540376100.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636206943540376100');
+
+ALTER TABLE `updates` CHANGE `state` `state` ENUM('RELEASED','CUSTOM','MODULE','ARCHIVED') CHARSET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT 'RELEASED' NOT NULL COMMENT 'defines if an update is released or archived.';
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1636206943540376100.sql:3]:
```suggestion
ALTER TABLE `updates` CHANGE `state` `state` ENUM('RELEASED','CUSTOM','MODULE','ARCHIVED') CHARSET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT 'RELEASED' NOT NULL COMMENT 'defines if an update is released, archived, custom or from a module.';
```

Tbh Module and custom is the same thing. But as I know you didn't implement it I will just suggest the comment update :D

Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1636206943540376100.sql:3]: @Kitzunu I'm just fixing the messup from here: https://github.com/azerothcore/azerothcore-wotlk/pull/8927#issuecomment-962452849 so I had to readd those files to fix the revision/db import script

@UltraNix could better address your suggestion

Comment (UltraNix) [reply data/sql/updates/pending_db_world/rev_1636206943540376100.sql:3]: Module and custom is now different.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1636206943540376100.sql:3]: ok

**Pull request merged**